src/pages/apply/FormPage.js

```diff
@@ -31,274 +32,554 @@ const FormPage = () => {
+              occupation: "",
+              twitterHandle: "",
+              instagramHandle: "",
+              facebookHandle: "",
+              linkedinHandle: "",
+              publicFigures: "",
+              mostAdmiredNGO: "",
+              preferredSocialMedia: "",
+              volunteerType: "",
+              interestedVolunteerUnits: [],
+              reasonForVolunteering: "",
+            }}
+            onSubmit={(values) => {
+              const formattedData = {
+                firstName: values.firstName,
+                lastName: values.lastName,
+                email: values.email,
+                phoneNumber: values.phoneNumber,
+                additionalData: {
+                  physicalAddress: values.physicalAddress,
+                  pronoun: values.pronoun,
+                  birthDay: values.birthDay,
+                  occupation: values.occupation,
+                },
+                socialBackground: {
+                  socialMediaHandles: {
+                    twitterHandle: values.twitterHandle,
+                    instagramHandle: values.instagramHandle,
+                    facebookHandle: values.facebookHandle,
+                    linkedinHandle: values.linkedinHandle,
+                  },
+                  publicFigures: values.publicFigures,
+                  mostAdmiredNGO: values.mostAdmiredNGO,
+                },
+                interests: {
+                  preferedSocialMedia: values.preferredSocialMedia,
+                  volunteerType: values.volunteerType,
+                  interestedVolunteerUnits: values.interestedVolunteerUnits,
+                },
+                reasonForVolunteering: values.reasonForVolunteering,
+              };
+              fetch(
+                "https://nhf-api.herokuapp.com/api/v1/volunteer-applicants",
+                {
+                  method: "POST",
+                  headers: { "Content-Type": "application/json" },
+                  body: JSON.stringify(formattedData),
+                }
+              )
+                .then((response) => {
+                  if (response.status === 201) {
+                    Swal.fire({
+                      title: "Application Successful!",
+                      text: "Check your email for the next steps.",
+                      icon: "success",
+                    });
+                    setTimeout(() => {
+                      window.location = "https://nhfafrica.org";
+                    }, 3000);
+                  } else if (response.status === 409) {
+                    Swal.fire({
+                      title: "Already Registered",
+                      text: "You have already registered",
+                      icon: "warning",
+                    });
+                    setTimeout(() => {
+                      window.location = "https://nhfafrica.org";
+                    }, 3000);
+                  }
+                })
+                .then((responseData) => {
```

Comment:
responseData here is the same as in the first .then()

Reply:
Thanks! This has been resolved.